### ISSUE TITLE
SNAPSHOT of 20210525: Dashboard not working

### ISSUE THREAD

**Issue by xatapult** [eXist-db/dashboard#180]
**Describe the bug**
In the 5.3.0 SNAPSHOT of 20210525 the Dashboard is not working.

**To Reproduce**
Install with this snapshot. Try to view the Dashboard.
JAR name: `exist-installer-5.3.0-SNAPSHOT+20210525025502.jar`

**Context (please always complete the following information):**
 - OS: Windows 10 but also on Linux
 - eXist-db version: 5.3.0 SNAPSHOT version of 20210525
 - Java Version:  various, one of them reports:
 ```
openjdk version "1.8.0_292"
OpenJDK Runtime Environment Corretto-8.292.10.1 (build 1.8.0_292-b10)
OpenJDK 64-Bit Server VM Corretto-8.292.10.1 (build 25.292-b10, mixed mode)
```

**Additional context**
- How is eXist-db installed? JAR
- Any custom changes: not yet

**Comment by joewiz** [comment]
The sudden appearance of this problem is due to https://github.com/eXist-db/exist/pull/3896, which caused `repo:get-resource` to throw an error if the 2nd argument, `$resource`, does not exist in the package (so now the function conforms to its function documentation, at least in this respect). 

It's surprising that the tests for https://github.com/eXist-db/exist/pull/3896 passed. We must be lacking an integration test that would open dashboard and open all resources that dashboard's landing page does. If we had such a test, we'd have seen that the request to http://localhost:8080/exist/apps/packageservice/packages/apps returns a 400 error, with this response:

```xml
<exception><path>/db/apps/packageservice/modules/local-apps.xql</path><message>err:FODC0002 Resource icon.svg does not exist. [at line 148, column 18, source: /db/apps/packageservice/modules/packages.xqm]
In function:
	(item()*, item()*, item()*) [148:18:/db/apps/packageservice/modules/packages.xqm]
	packages:scan-repo(function(*)) [147:5:/db/apps/packageservice/modules/packages.xqm]
	packages:installed-apps(xs:string) [118:19:/db/apps/packageservice/modules/packages.xqm]
	packages:get-local(xs:string) [38:5:/db/apps/packageservice/modules/packages.xqm]
	packages:get-local-applications() [16:6:/db/apps/packageservice/modules/packages.xqm]</message></exception>
```

The error is thrown in /db/apps/packageservice/modules/packages.xqm on line 167-8 where `repo:get-resource` is called: https://github.com/eXist-db/existdb-packageservice/blob/master/modules/packages.xqm#L167-L168

In other words, this is a packageservice issue, not a dashboard issue.

Now that `repo:get-resource` throws an error, we really need a `repo:resource-available` function.
